assignments/meetup.md

```diff
@@ -55,4 +55,5 @@ Schedule
 Meeting                     | Meeting Date | Presentation Date | Students
 -------                     | :----------: | :---------------: | --------
 Placeholder                 | Placeholer   | Placeholder       | Placeholder
+Denver Gophers                 | September 28th   |        | Adrien Perinet, Austin Schussler, Ian Tobiason
```

Comment:
Hey, great job but one small change -- post the date as 9/28 instead of September 28th. Otherwise looks good!